data/data/google/network/variables.tf

```diff
@@ -0,0 +1,15 @@
+variable "cluster_id" {
+  type = "string"
```

Comment:
this are pre 0.12 terraform ways of defining the variable. needs to be `type = string`